### PR TITLE
feat: home

### PR DESCRIPTION
**Pull request by dev-ashishsethi**
image

### CODE REVIEW

File: src/Components/Nav/Nav.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          </div>
+        </button>
+      ) : (
+        <Link to="/SignIn" className="login-btn">
+          {" "}
+          <button className="btn btn-primary login-btn" onClick={LoginHandler}>
+            Log In
+          </button>
+        </Link>
+      )} */}
```
Comment (m26unkwn) [inline src/Components/Nav/Nav.jsx:34]: remove this comment

File: src/Pages/Ques/Ques.jsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    shuffleArray(questions?.incorrect_answers);
+  }, [index]);
+
+  function shuffleArray(array) {
+    for (let i = array?.length - 1; i > 0; i--) {
+      const j = Math.floor(Math.random() * (i + 1));
+      [array[i], array[j]] = [array[j], array[i]];
+    }
+    return array;
+  }
```
Comment (m26unkwn) [inline src/Pages/Ques/Ques.jsx:59]: you can take this function out of the component in util function

**Review by m26unkwn — APPROVED** [review]
You can go ahead with the code after the suggested changes.

Code is clean and UI is awesome.

File: src/Components/Card/Card.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  width: 22rem;
+  border-radius: 8px;
+  background-color: var(--grey);
+}
+a{
+    background-color: transparent;
+}
+.quiz-card:hover{
+    transform: scale(1.03);
+}
```
Comment (PuneetP16) [inline src/Components/Card/Card.css:12]: Suggestion: Format the CSS properly. Give appropriate spacing.

File: src/Components/Nav/Nav.jsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        </p>
+      </Link>
+
+{/*       
+      {login ? (
+        <button className="profile-btn">
+          <All.IconoirProfileCircled className="profile" />
+
+          <div className="profile-options">
+            <ul>
```
Comment (PuneetP16) [inline src/Components/Nav/Nav.jsx:19]: SuggestIon: remove commented code.

File: src/Pages/Ques/Ques.css [hunk 5]
```diff
@@ -0,0 +1,29 @@
+.question {
+  display: flex;
+  justify-content: space-evenly;
+  height: 100vh;
+  flex-direction: column;
+  margin-inline: 20rem;
+}
+.options {
+  display: flex;
+  list-style: none;
```
Comment (PuneetP16) [inline src/Pages/Ques/Ques.css:10]: Here too.

**Review by PuneetP16 — APPROVED** [review]
Your UI is quite awesome. Good work on logic, Good to merge. Check the comments mentioned.

**Pull request merged**